app/services/search/book_versions/i1/indexing_strategy.rb

```diff
@@ -27,7 +27,7 @@ def index_metadata
     end
 
     def index(book:, index_name:)
-      Rails.logger.info("OpenSearch: indexing #{index_name} with #{book.root_book_part.pages.count} pages")
+      Rails.logger.info("I1::IndexingStrategy: Creating index #{index_name} with #{book.root_book_part.pages.count} pages")
```

Comment:
Just noticed that the `pages` method isn't memoized.  Can you make that change in cnx-ruby?